userspace/libpman/src/events_prog_table.h

```diff
@@ -24,6 +24,7 @@ limitations under the License.
 
 typedef struct {
 	char* name;
+	enum bpf_prog_type prog_type;
```

Comment:
Add the program type information, because now programs in `userspace/libpman/src/events_prog_table.c` can have two types: `BPF_PROG_TYPE_RAW_TRACEPOINT` (as before) and `BPF_PROG_TYPE_TRACEPOINT` (new TTM mitigation programs). This information is used in https://github.com/falcosecurity/libs/pull/2576/files#diff-da696150f59c988d4ffde0c0a853f1d258c0db247a2be3f543ed945bdf734603R45